Shared/Bytes.swift

```diff
@@ -10,19 +10,15 @@ public typealias GUID = String
  * Utilities for futzing with bytes and such.
  */
 open class Bytes {
-    open class func generateRandomBytes(_ len: UInt) -> Data {
-        let len = Int(len)
-        var data = Data(count: len)
-        data.withUnsafeMutableBytes { (p: UnsafeMutablePointer<UInt8>) in
-            assert(SecRandomCopyBytes(kSecRandomDefault, len, p) == 0, "Random byte generation failed.")
-        }
-        return data
+    open class func generateRandomBytes(_ len: Int) -> Data {
```

Comment:
nit: I think this made more sense the way it was as a `UInt` so you wouldn't have to cast in Profile.swift